pom.xml

```diff
@@ -775,6 +775,38 @@
           <artifactId>versions-maven-plugin</artifactId>
           <version>2.7</version>
         </plugin>
+        <plugin>
+          <groupId>com.mycila</groupId>
+          <artifactId>license-maven-plugin</artifactId>
+          <version>3.0</version>
+          <configuration>
+            <header>${session.executionRootDirectory}/contrib/license-header.txt</header>
+            <excludes combine.children="append">
+              <exclude>**/DEPENDENCIES</exclude>
+              <exclude>**/LICENSE</exclude>
+              <exclude>**/NOTICE</exclude>
+              <exclude>**/target/**</exclude>
+              <exclude>contrib/javadoc11.patch</exclude>
+            </excludes>
+            <mapping>
+              <HelloWorldTemplate>SLASHSTAR_STYLE</HelloWorldTemplate>
+              <Makefile>SCRIPT_STYLE</Makefile>
+              <TestTemplate>SLASHSTAR_STYLE</TestTemplate>
+              <accumulo-cluster>SCRIPT_STYLE</accumulo-cluster>
```

Comment:
Is this setting a mapping for a maven module?

Reply:
These mappings apply to specific files that appear in some of our maven modules. I could move these mappings to module-specific pom.xml configuration for the plugin, but it won't change the behavior of the plugin with respect to these files.

Reply:
My question didn't even make sense, however I still found your answer useful.  I was confusing Accumulo with Fluo, Fluo had a fluo-cluster maven module.  Accumulo has no such module.  

I see the following.

```
$ find . -name "*accumulo-cluster*"
./assemble/bin/accumulo-cluster
```